### PR TITLE
Updates to courses and OMSA

### PR DESCRIPTION
**Pull request by driscoll42**
This is to resolve bugs https://github.com/OMSCentral/omscentral/issues/209 And https://github.com/OMSCentral/omscentral/issues/205

### CODE REVIEW

File: server/database/data/courses.ts [hunk 1]
```diff
@@ -940,6 +940,14 @@ export const courses: PMO<Course>[] = [
...
     foundational: false,
     deprecated: false,
+  },
+    id: 'MGT-8833',
+    department: 'MGT',
+    number: '8833',
+    name: 'Privacy for Professionals',
+    aliases: [],
+    foundational: false,
+    deprecated: false,
```
Comment (mehmetbajin) [inline server/database/data/courses.ts:950]: Given there's already a Privacy for Professionals course, what is the motivation for adding this one?

File: server/database/data/courses.ts [hunk 2]
```diff
@@ -996,4 +1004,14 @@ export const courses: PMO<Course>[] = [
...
+  {
+    id: 'VIP-6600',
+    department: 'VIP',
+    number: '6600',
+    name: 'Vertically Integrated Projects',
+    aliases: ['VIP'],
+    foundational: false,
+    deprecated: false,
+    link: 'https://www.vip.gatech.edu/',
+  },
```
Comment (mehmetbajin) [inline server/database/data/courses.ts:1016]: Can you provide more context on VIP-6600? The link is unlike the other courses.

File: server/database/data/specializations.ts [hunk 3]
```diff
@@ -168,6 +168,7 @@ export const specializations: PMO<Specialization>[] = [
           'CS-6460',
           'CS-6465',
           'CS-6471',
+		  'CS-6603',
```
Comment (mehmetbajin) [inline server/database/data/specializations.ts:171]: Please run `npm run format` (inside `./server` director) to ensure the whitespace is properly formatted.

**Review by mehmetbajin — CHANGES_REQUESTED** [review]

File: server/database/data/courses.ts (hunk 2 above)
Comment (driscoll42) [reply server/database/data/courses.ts:1016]: It is different from all the other courses. VIP (Vertically Integrated Projects) are not normal courses with lectures/homework/projects/exams, instead they are multidisciplinary projects you work on for credit lead by faculty. It's more like a typical work project, where you meet weekly as a group with the faculty member to discuss progress, problems, next steps, etc... 

In the case of the one I'm in, it's part of the Ivan Allen Digital Archive (http://allenarchive.iac.gatech.edu/) where we are working on making the website more accessible/usable, specifically for myself I am working on linking all the tags on the documents to Wikipedia articles, as well as improving the quality of the tags, with a combination of NLP/NER and some models trained to detect appropriate articles for tags.

Personally I think having this "course" on OMSCentral would be highly valuable as there's relatively little visibility into the program, hard to find people with experience in it, and having it on OMSCentral would help draw attention to it, let people talk about their time in it, and it is a course you get credit for on OMSCS, like any of the other traditional ones.

File: server/database/data/courses.ts (hunk 1 above)
Comment (driscoll42) [reply server/database/data/courses.ts:950]: I was searching for the number, not the title, looks like the number has changed, I'll remove this one.

File: server/database/data/specializations.ts (hunk 3 above)
Comment (driscoll42) [reply server/database/data/specializations.ts]: I think it's corrected now

**Review by mehmetbajin — APPROVED** [review]

**Pull request merged**